### ISSUE TITLE
Tab error

### ISSUE THREAD

**Issue by sashank** [m4ll0k/takeover#7]
```
python3 takeover.py --sub-domain-list input.txt --set-output sub_out.txt
  File "takeover.py", line 131
    print(" _( ^)  |T|A|K|E|O|V|E|R|   ")
                                        ^
TabError: inconsistent use of tabs and spaces in indentation
```
Am getting the above error please help

**Comment by codingo** [comment]
Hi @sashank, I'm on mobile at the moment but I've attempted to fix this in #8. Could you please test this and confirm on the pull request if this is resolved so it can be landeed to master?

**Comment by sashank** [comment]
Is this fixed ?
